sisl/viz/backends/plotly/backend.py

```diff
@@ -1,7 +1,6 @@
 from collections import defaultdict
 import itertools
 from functools import partial
-from sisl.viz.plots.tests.test_bands import backend
```

Comment:
Ok, I see it was a typo. ;)

Reply:
Yeah it was automatically imported by VS studio. Text editors are so smart nowadays :eyes: hahaha